Lecture_18/Homework_18.py

```diff
@@ -0,0 +1,32 @@
+class Bot:
+    def __init__(self, name,):
+        self.name = name
+    def say_name(self, name):
+        return Bot(self.say_name)
+    def send_message(self, message):
+        return Bot(self.send_message())
+
+
+class TelegramBot(Bot):
+    def __init__(self, url, chat_id):
```

Comment:
I think the task was to inherit `init` method
```suggestion
    def __init__(self, name, url=None, chat_id=None):
        super().__init__(name)
```